portfolio/src/main/webapp/js/script.js

```diff
@@ -251,3 +266,19 @@ function instanciateLoadMoreButton() {
     button.onclick = loadQuestions;
     buttonContainer.appendChild(button);
 }
+// This function triggers the UploadUrl servlet
+function fetchBlobstoreUrl() {
```

Comment:
is there a time limit on the upload URL? If the user leaves the browser open for a long time, will it expire?

Reply:
Interesting! I have no idea but I have tried leaving it for more than 45 minutes and it worked!